core/common/src/main/java/alluxio/master/PollingMasterInquireClient.java

```diff
@@ -131,7 +131,8 @@ private InetSocketAddress getAddress() {
   private void pingMetaService(InetSocketAddress address) throws AlluxioStatusException {
     GrpcChannel channel =
         GrpcChannelBuilder.newBuilder(GrpcServerAddress.create(address), mConfiguration)
-            .setSubject(mUserState.getSubject()).setClientType("MasterInquireClient").build();
+            .setSubject(mUserState.getSubject()).setClientType("MasterInquireClient")
+            .disableAuthentication().build();
```

Comment:
Please add a comment that we are disabling authentication on purpose.

Reply:
Added.